src/pages/index.tsx

```diff
@@ -57,19 +69,24 @@ const HomePage: NextPageWithLayout = () => {
       const { showOnTS } = getNotificationLocalStorage() || {};
 
       if ((tosAccepted && !showOnTS) || (tosAccepted && showOnTS < Date.now())) {
-        setTimeout(() => {
-          setShowNotificationModalState(showNotificationModal());
-        }, 10000);
+        // let's test it on preview first and then I can change that
+        if (iOSDevice && !isHomeScreenApp) {
+          setIosHomeScreenPrompt(true);
+        } else {
+          setTimeout(() => {
+            setShowNotificationModalState(showNotificationModal());
+          }, 10000);
+        }
```

Comment:
As I mention in my loom video we can change this

Reply:
This is 10 seconds?  What happens if the user clicks to a different screen before 10 seconds has passed?  I'm thinking perhaps this should be 5 seconds.

Reply:
3 👍🏾